src/2.0/__tests__/e2e.test.ts

```diff
@@ -0,0 +1,181 @@
+import { ProofType, SchemaId, sign, validateSchema, verifySignature, wrapDocument, wrapDocuments } from "../..";
```

Comment:
Add this comment at the wrong place: Need to think about how to correctly handle OA2018 proof with DID proof :)